docs/application/native/guides/text-input/tts.md

```diff
@@ -686,6 +719,38 @@ To start, pause, and stop the playback, follow the steps below:
   }
   ```
 
+<a name="playing_mode"></a>
+## Playing Mode
+
+Playing mode determines which process, either TTS service or TTS client, plays the synthesized PCM data. There're 2 different playing modes. They are described below:
+
+> [!NOTE]
+> Playing mode is different from TTS mode. Playing mode only determines a subject to play PCM data.
+
+- `TTS_PLAYING_MODE_BY_SERVICE`: TTS service plays the synthesized PCM data. If you do not set playing mode, it will be set as `TTS_PLAYING_MODE_BY_SERVICE`.
+- `TTS_PLAYING_MODE_BY_CLIENT`: TTS client receives the synthesized PCM data from TTS service, and plays it directly.
+
+To decide between the client-side playback mode and the service-side playback mode, follow the steps below:
+
+- Set the playing mode
+
+  If the application wants to play the PCM data directly instead of playing it that the application requested for synthesis in the TTS service, you can set the playback mode to `TTS_PLAYING_MODE_BY_CLIENT`. If not set, because the default value is `TTS_PLAYING_MODE_BY_SERVICE`, the TTS service will play it when the TTS client requests to synthesize the text.
+
+  > [!NOTE]
+  > If the playing mode is `TTS_PLAYING_MODE_BY_SERVICE`, you don't need to set callback. It will be played automatically by the TTS service. If you set the mode to `TTS_PLAYING_MODE_BY_CLIENT`, you can receive the synthesized PCM data via `synthesized_pcm_callback`. And you only change playing mode when TTS client state is `TTS_STATE_CREATED`. If you want to use both playing mode, it would be better to create 2 TTS handles.
```

Comment:
```suggestion
  > If the playing mode is `TTS_PLAYING_MODE_BY_SERVICE`, you don't need to set a callback. It will be played automatically by the TTS service. If you set the mode to `TTS_PLAYING_MODE_BY_CLIENT`, you can receive the synthesized PCM data via `synthesized_pcm_callback` and only change playing mode when TTS client state is `TTS_STATE_CREATED`. If you want to use both playing modes, it would be better to create 2 TTS handles:
```

Reply:
Thank you for fixing it.
I'll change it according to your comment.